lib/Onyx.js

```diff
@@ -1604,6 +1605,7 @@ function subscribeToEvents() {
  * @param {Boolean} [options.shouldSyncMultipleInstances] Auto synchronize storage events between multiple instances
  * of Onyx running in different tabs/windows. Defaults to true for platforms that support local storage (web/desktop)
  * @param {Boolean} [options.debugSetState] Enables debugging setState() calls to connected components.
+ * @returns {Boolean} Whether there was a problem initializing the platform storage provider and Onyx uses the fallback provider instead.
```

Comment:
What will we use this for?

Reply:
To indicate to the user, that the app has degraded performance and is not using storage. Maybe, to also react to this or adapt the app...

> It might be nice to have a hybrid approach where Onyx is only using memory storage, and then the App displays a global banner like "There could be degraded performance, blah blah" to let the user know that something isn't quite right.

From https://github.com/Expensify/App/issues/29403#issuecomment-1821216687

Reply:
Ah, got it thanks! We should pick one way to do this. Either have it get returned by `Onyx.init()` or expose some method like you are doing with `isUsingFallbackStorageProvider()`